pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix

```diff
@@ -117,6 +117,8 @@ stdenv.mkDerivation rec {
 
   src = srcs.${stdenv.hostPlatform.system} or (throw "unsupported system: ${stdenv.hostPlatform.system}");
 
+  # don't bother checking cache.nixos.org because it isn't built or cached there
+  # (see meta.hydraPlatforms)
   preferLocalBuild = true;
   allowSubstitutes = false;
```

Comment:
This is only true for the wrapper which indeed can be faster build locally and does not necessarily need to be cached on hydra but we want to remove meta.hydraPlatforms to cache the src tarball.